pkg/controller/node/nodecontroller.go

```diff
@@ -215,7 +215,7 @@ func NewNodeController(
 	runTaintManager bool,
 	useTaintBasedEvictions bool) (*NodeController, error) {
 	eventBroadcaster := record.NewBroadcaster()
-	recorder := eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: "controllermanager"})
+	recorder := eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: "node-controller"})
```

Comment:
It makes sense. Would you please check EventSource of other controllers in pkg/controller ?